tests/e2e/distributed_queries/awesome_memgraph_functions.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2022 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+import sys
+import time
+import typing
+
+import mgclient
+import pytest
+
+from common import *
+
+
+def test_vertex_creation_and_scanall(connection):
+    wait_for_shard_manager_to_initialize()
+    cursor = connection.cursor()
+
+    assert has_n_result_row(cursor, "CREATE (n :label {property:1})", 0)
```

Comment:
@kostasrim since e.g. `VertexAccessor` returns 0 for `In/OutDegree`, I think it makes sense in this test to add real test cases for each function but somehow disable them. As we move on we'll implement some of these functionalities + when we reach the point where the release will be close, we just enable all the tests and observe/fix what is not working 🤔 

What do you think about that idea?

Reply:
No, no, in this case, it wouldn't be commented code, it would be legit test cases just not executed in the CI (runtime config for these tests). I understand these two are very very similar, but I would like to avoid the situation to miss some of the required functionality, I think these e2e tests are a perfect spot to ensure we don't miss functionality. I agree QE work is more important, but since we are now covering these functions, we/you have the whole context, later the whole thing we'll have to be planned again, it's very likely that we'll miss something, etc...

Reply:
I decided not to add more tests because:
1. I think the current functionality is tested well.
2. I would not like to have disabled tests as any potential change might require modifications to the disabled tests which requires the reviewer or the PR implemented to recall that such disabled tests exist.